src/tile_map.vert

```diff
@@ -20,7 +20,7 @@ layout(set = 2, binding = 1) uniform TileMapChunk {
 
 void main() {
     v_Uv = Vertex_Uv;
-    vec3 position = Vertex_Position * vec3(5.0, 5.0, 1.0);
+    vec3 position = Vertex_Position * vec3(4.0, 4.0, 1.0);
```

Comment:
This is some arbitrary scale I came up with. We should likely remove this so that sprite sheet pixels match up with the screen's pixels and instead use camera for zoom.

Reply:
That's fair. Currently, zooming in doesn't work in bevy according to https://github.com/bevyengine/bevy/issues/239, so leaving it as it is until the issue is fixed in bevy might be better.

Reply:
hmm, yeah good point lets leave it for now!